### PR TITLE
Startup roscore and bringup inside tmux sessions

### PR DESCRIPTION
**Pull request by ipa-bnm**
This PR partly solves https://github.com/ipa320/msh/issues/628

### CODE REVIEW

**Comment by ipa-bnm** [comment]
@ipa-nhg I've changed mydistro and myrobotname. Please check them again because the ROBOT env was set to a different value.

**Comment by ipa-nhg** [comment]
@ipa-bnm thanks, actually now the ROBOT variable and myrobotname (from hostname) should be always the same), I will check it!
Please merge this https://github.com/ipa-bnm/setup_cob4/pull/3 to update also the install script. Thanks!

**Comment by ipa-fxm** [comment]
fix https://github.com/ipa320/msh/issues/628

File: upstart_msh/cob-start [hunk 1]
```diff
@@ -92,9 +77,28 @@ export ROS_IP=`rosrun robot_upstart getifip eth0`
...
+  echo "starting VM..."
+  su mimic -c "ssh myrobotname-h1 'export DISPLAY=:0 && virtualbox --startvm myrobotname-win --fullscreen'"&
+fi
+
+
+#start bringup inside tmux session
+su $USER -l -c "tmux new -d -s bringup"
+su $USER -l -c "tmux send -t bringup 'sh /etc/ros/mydistro/cob.d/robot_launch.sh' ENTER"
+
+ssh $USER@myrobotname-t1 "tmux new -d -s statemachine"
```
Comment (ipa-fmw) [inline upstart_msh/cob-start]: needs to be started on t2! see https://github.com/ipa320/msh/blob/indigo_dev/msh_bringup/launch/application.xml#L97

File: upstart_msh/cob-stop [hunk 2]
```diff
@@ -26,18 +26,29 @@
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 # 
 # Please send comments, questions, or patches to code@clearpathrobotics.com 
-if [ -e /tmp/robot.launch.pid ]; then
+ROBOT=myrobotname
+USER=myuser
+
+if (ssh $USER@$ROBOT-t1 "tmux has-session -t 'statemachine' 2> /dev/null"); then
```
Comment (ipa-fmw) [inline upstart_msh/cob-stop]: is running on -t2

File: upstart_msh/cob-stop [hunk 3]
```diff
@@ -52,6 +63,9 @@ else
   echo "......(cob-start not running)....."
 fi
 
+#cleaning up roscore
+su $USER -l -c "rosnode cleanup"
```
Comment (ipa-fmw) [inline upstart_msh/cob-stop]: this throws an error: `-su: rosnode: command not found`

can be fixed by 
```
su $USER -l -c "source /opt/ros/indigo/setup.bash && rosnode cleanup"
```

probably replace `indigo` by `mydistro`

Comment (ipa-fmw) [inline upstart_msh/cob-stop]: fixed in https://github.com/ipa-bnm/setup_cob4/pull/4

File: upstart_msh/cob-start (hunk 1 above)
Comment (ipa-fmw) [inline upstart_msh/cob-start]: fixed in https://github.com/ipa-bnm/setup_cob4/pull/4

File: upstart_msh/cob-stop (hunk 2 above)
Comment (ipa-fmw) [inline upstart_msh/cob-stop]: fixed in https://github.com/ipa-bnm/setup_cob4/pull/4

**Comment by ipa-bnm** [comment]
@ipa-fxm thx for your corrections

**Pull request merged**